msal/src/main/java/com/microsoft/identity/client/internal/CommandParametersAdapter.java

```diff
@@ -203,6 +206,7 @@ public static InteractiveTokenCommandParameters createInteractiveTokenCommandPar
                 .powerOptCheckEnabled(configuration.isPowerOptCheckForEnabled())
                 .correlationId(parameters.getCorrelationId())
                 .preferredAuthMethod(parameters.getPreferredAuthMethod())
+                .requestHeaders(addPasskeyHeader(parameters.getExtraQueryStringParameters(), configuration))
```

Comment:
kinda of a nit: I think parameters.getExtraQueryStringParameters() is going to be returning the original set of query string paramters and not the updated one which would be generated by appendToExtraQueryParametersIfWebAuthnCapable(parameters.getExtraQueryStringParameters(), configuration). I think based on the logic below, it potentially would be possible that the header would be added when it shouldn't be (like in the case where Android version <= 9 and the qs param should be removed.

Maybe call the appendToExtraQueryParameterIfWebAuthnCapable method earlier and save it to a variable?